graylog2-web-interface/src/views/components/widgets/WidgetFailed.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow strict
+import * as React from 'react';
+
+import { Icon } from 'components/common';
+import styles from './MessageWidgets.css';
+
+type Props = {
+  error: Error,
+};
+const WidgetFailed = ({ error }: Props) => (
+  <div className={styles.spinnerContainer}>
+    <Icon name="exclamation-triangle" size="3x" className={styles.iconMargin} />
+    <span>
+      <strong>While rendering this widget, the following error occurred:</strong>
+
+      <p>{error.toString()}</p>
+    </span>
+  </div>
+);
```

Comment:
We could use the `ErrorWidget` component for the Icon and text style here. It has the same layout, but the component would need some adjustments, like a prop to provide a custom headline.